thrift/linkattacher/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+# Copyright (C) 2016 Istituto Italiano di Tecnologia iCub Facility & RBCS Department
+# Authors: Yeshasvi Tirupachuri
+# CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+
+cmake_minimum_required(VERSION 2.8.11)
+
+set(LIBRARY_TARGET_NAME gazebo_yarp_rpc_linkattacher)
+
+yarp_idl_to_dir(linkattacher_rpc.thrift ${CMAKE_CURRENT_SOURCE_DIR}/autogenerated LARPC_SRC LARPC_HEADERS LARPC_INCLUDES)
+
+include_directories(${LARPC_INCLUDES} ${YARP_INCLUDE_DIRS})
+add_library(${LIBRARY_TARGET_NAME} SHARED ${LARPC_SRC} ${LARPC_HEADERS})
+target_link_libraries(${LIBRARY_TARGET_NAME} ${YARP_LIBRARIES})
+
+set_target_properties(${LIBRARY_TARGET_NAME} PROPERTIES VERSION ${${PROJECT_NAME}_VERSION}
+                                                        PUBLIC_HEADER "${LARPC_HEADERS}")
+
+target_include_directories(${LIBRARY_TARGET_NAME} PUBLIC
+                           "$<BUILD_INTERFACE:${LARPC_INCLUDES}>"
+                           "$<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/gazebo>")
+
+set_property(TARGET ${LIBRARY_TARGET_NAME} PROPERTY PUBLIC_HEADER ${LARPC_HEADERS})
```

Comment:
Check that the headers are installed. If I recall well if `${LARPC_HEADERS}` are more than one you should use apices

```cmake
set_property(TARGET ${LIBRARY_TARGET_NAME} PROPERTY PUBLIC_HEADER "${LARPC_HEADERS}")
```

Otherwise only the first one is installed. This has to be checked.

Reply:
@diegoferigo  I checked in the path `~/robotology-superbuild/build/install/include/gazebo` and find `LinkAttacherServer.h` thrift autogenerated header already installed correctly. This is the only file we need right? I do not follow your comment.

Reply:
I am not aware of it. Anyway, I think in this case it is not relevant I guess.

Reply:
Currently no. Though, if in the future a new header is added there's the possibility that it will not get installed.

Reply:
Should I change it for this plugin ?? @diegoferigo

Reply:
To avoid mistakes in the future, just add the double apices.

Reply:
Done :grinning: